Fun Commands/Mini-Games/Dyno-RPG/infect.txt

```diff
@@ -0,0 +1,5 @@
+{delete}
+{silent}
+{require:{&Adventurer}}
+{!role $1 +Infected}
+$1 was infected by {user}!
```

Comment:
May be useless, but an idea for continuity:
```suggestion
$1 was infected by {user.username}!
```

Reply:
That command's response has actually been reworked to removed the user mention =) The new push should display it =)